src/app/code/community/Shopgate/Cloudapi/etc/config.xml

```diff
@@ -2,7 +2,7 @@
 <config>
     <modules>
         <Shopgate_Cloudapi>
-            <version>0.1.1</version>
+            <version>0.1.2</version>
```

Comment:
can you remove the previous upgrade script and change file name to upgrade from 1.0. The reason for that is that the previous upgrade script does the same thing as the install script, since we had not launched yet, we can remove that file.